README.md

```diff
@@ -1,4 +1,8 @@
-## Overview
```

Comment:
the Table of contents is outdated with this removed header

Reply:
good catch